package-lock.json

```diff
@@ -20050,15 +20966,6 @@
       "integrity": "sha512-O3sR1/opvCDGLEVcvrGTMtLac8GJ5IwZC4puPrLuRj3l7ICKvkmA0vGuU9OW8mV9WIBRnaxp5GJh9IEAaNOoYg==",
       "dev": true
     },
-    "node_modules/immutable": {
-      "version": "3.7.6",
-      "resolved": "https://registry.npmjs.org/immutable/-/immutable-3.7.6.tgz",
-      "integrity": "sha1-E7TTyxK++hVIKib+Gy665kAHHks=",
-      "peer": true,
-      "engines": {
-        "node": ">=0.8.0"
-      }
-    },
```

Comment:
IDK why this gets removed from time to time, but undoing this here should fix some of the errors here.